rfps/multi-chain-block-explorer.md

```diff
@@ -0,0 +1,12 @@
+# Multi-chain Block Explorer
+
+* **Status:** Open
+
+## Project Description :page_facing_up: 
+
+As parachains become an integral part of the Polkadot and Kusama ecosystems, a cross-chain block & accounts explorer becomes all the more useful.
+
+Some of the functionality that should be covered as part of the development:
+- select which chains to view (e.g. default when selecting Kusama is Kusama relay + all its parachains). Should be feasible to select alternative mainnets too.
+- input an address and see Tx's on all the selected mainnets, including teleports/XCMs between various parachains
+- when a Tx includes a XCM, it should be easy and intuitive to open the relevant block from the other chain(s).
```

Comment:
Honestly [Subscan](https://statemine.subscan.io/) should be well positioned to implement all of these soon, I bet it's on their roadmap already. It may make sense to reach out to them.

Reply:
Also for points 1 and 2 I'm a bit afraid this would encourage reusing the same address across chains, which as far as I'm aware we don't actually expect people to do.

Reply:
Talked to Joe yesterday and this is apparently going to change very soon. Reusing the same address is expected as one should be able to maintain the same identity across chains - the big ugly warning on Polkadot.js only exists because of ledger using different derivation paths on Kusama vs Polkadot and people getting their funds trapped as a consequence, and will be removed at some point.